mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -1538,7 +1538,7 @@ def _compute_moc(latBins, nz, latCell, regionCellMask, transportZ,
     """compute meridionally integrated MOC streamfunction"""
 
     mocTop = np.zeros([np.size(latBins), nz + 1])
-    mocTop[0, range(1, nz + 1)] = transportZ.cumsum()
+    mocTop[0, nz:0:-1] = - transportZ[nz-1:-1:-1].cumsum()
```

Comment:
@vanroekel, I was trying to implement what I thought would be in https://github.com/E3SM-Project/E3SM/pull/5170.  As I stated in https://github.com/E3SM-Project/E3SM/pull/5170#pullrequestreview-1099712156, I don't think things are quite right in that PR (and also not here).

I think we need `mocTop[0, nz] = 0.` and then sum the transport up from there.  I will fix that here.